src/ngraph/op/tensor_iterator.cpp

```diff
@@ -38,6 +38,11 @@ op::TensorIterator::TensorIterator(const OutputVector& values)
 {
 }
 
+bool ngraph::op::TensorIterator::visit_attributes(AttributeVisitor& visitor)
+{
+    return true;
```

Comment:
How should I set/get Body of TI? If I create a TI as an abstract layer how can I create it?

Reply:
Since this operator is misalignment with the spec I will remove it from this PR.
TensorIterator should be adjusted in another PR.